programs/ssl/ssl_server2.c

```diff
@@ -2146,14 +2155,18 @@ int main(int argc, char *argv[])
             if (opt.ticket_timeout < 0) {
                 goto usage;
             }
-        } else if (strcmp(p, "ticket_aead") == 0) {
```

Comment:
Any reason to not leave the else with the if? There is a check right below this that does it this way, and it avoids having to split up the else and the if, which I think makes it a little harder to read.